.github/workflows/i18n_update.yaml

```diff
@@ -50,4 +50,62 @@ jobs:
         env:
           PR_URL: ${{ steps.cpr.outputs.pull-request-url }}
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
- 
\ No newline at end of file
+  get_release_branches: 
+    name: Get Unreleased Branches
+    runs-on: ubuntu-latest
+    outputs:
+      matrix: ${{ steps.gen.outputs.branches }}
+    steps:
+      - name: Clone repository
+        uses: actions/checkout@v3
+        with:
+          submodules: 'true'
+      - name: Generate Branch List
+        id: gen
+        shell: pwsh
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          ./.github/scripts/get_unreleased_branches.ps1  >> $env:GITHUB_OUTPUT
```

Comment:
I don't think the script is executable?

Reply:
Powershell does not really care about that 😅